### PR TITLE
Add @timlegge's timeline post

### PR DESCRIPTION
**Pull request by sjn**

### CODE REVIEW

File: _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+### CVE-2024-23525: Spreadsheet::ParseXLSX XML external entity attack vulnerability
+
+An Pham discovered a XML external entity injection (XXE) vulnerability in Spreadsheet::ParseXLSX version 0.29 and earlier, enabling an attacker to interact with the system 
+
+This is a classic XML external entity (XXE) injection vulnerability, in which the attacker can cause the vulnerable code to include data (or a file) that should not be available, by simply instructing the XML parser to load external data.
+The PoC also includes an example that would cause a DoS.
+
+Configuring an XML parser to allow loading external entities is dangerous and should never be the default.
+
+A detailed write up of the vulnerability and PoC is available at ttps://gist.github.com/phvietan/d1c95a88ab6e17047b0248d6bf9eac4a
```
Comment (timlegge) [inline _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md:46]: https is missing the h

File: _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+On January 18th, CPANSec notifies oss-security mailing list and Michael Daum released Spreadsheet::ParseXLSX version 0.31 which referenced CVE-2024-23525 which had been fixed in version 0.30.
+
+
+## Responsible Disclosure
+
+Based on the information provided by Đình Hải Lê, and publicly available information, it appears that Đình Hải Lê and/or other parties followed a responsible disclosure process.
+Attempts to contact the previous maintainers were followed by a reasonable waiting period before the PoC was published.
+After the PoC was published huntr.com also attempted to contact the previous maintainers.
+Resolution
```
Comment (timlegge) [inline _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md:118]: ## Resolution

**Review by timlegge — CHANGES_REQUESTED** [review]
Just a couple of changes

File: _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md (hunk 2 above)
Comment (sjn) [reply _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md]: fixed

File: _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md (hunk 1 above)
Comment (sjn) [reply _posts/2024-02-10-vulnerable-spreadsheet-parsing-modules.md]: fixed

**Pull request merged**